### PR TITLE
Style tweak to make links in ins/del more visible

### PR DESCRIPTION
**Pull request by frivoal**
Compatibility should be pretty good: https://caniuse.com/classlist
Manual replacements for classList.add and classList.remove can be written if necessary (but the default view will load fine anyway, even in ancient browsers).

### CODE REVIEW

File: src/base.css [hunk 1]
```diff
@@ -598,6 +598,14 @@
 		background: var(--ins-bg);
 		text-decoration: underline;
 	}
+	ins.diff-inactive,
+	del.diff-inactive {
+		all: unset;
+	}
+	ins:not(.diff-inactive) a,
+	del:not(.diff-inactive) a {
```
Comment (frivoal) [inline src/base.css]: Alternatively, maybe this should be:
```suggestion
	ins:not(.diff-inactive) *,
	del:not(.diff-inactive) * {
```
Links aren't the only things that change the color of the text, and all things things are hard to notice as being part of an ins or del if they're not green or red. As this goes away and the normal colors come back in the "current" and "future" views, I think the tradeoff is reasonable: in diff view, you most easily seen what has been added/removed, and in other views, you get the normal styling for better legibility.

**Review by deniak — COMMENTED** [review]
LGTM but I'll let @fantasai have the final word.

**Review by fantasai — APPROVED** [review]

**Comment by fantasai** [comment]
Let's give it a try. :) If we run into problems, we can make further tweaks.

**Comment by deniak** [comment]
Changes now deployed.

**Pull request merged**